global.pot

```diff
@@ -8268,7 +8268,7 @@ msgstr ""
 
 #: _protected/app/system/modules/user/forms/LoginForm.php:33
 #: _protected/app/system/modules/user/forms/LoginSplashForm.php:34
-msgid "Stay signed in"
+msgid "Запомнить меня"
```

Comment:
The .pot file is only English. It's the one without translation. Just the shall. You will need to start your translation from that one, and create a Russian folder (ru_RU) as well that will contain your translation files.

For further information, please see here: https://github.com/pH7Software/pH7-Internationalization#how-to-install-those-languages

Reply:
Hello,

I'm sorry I can't open a new post and here I really need help please.
I tried translating the script to French but it doesn't work.
I downloaded the translated files and put them in the correct folders (config.ini in the config folder, global.po and global.mo in the LC_MESSAGES folder and language.php in the general fr_FR folder.
I then selected the language in the script but there is no change.
Have I forgotten something?

Thank you in advance for your reply.